src/main/java/com/minecolonies/coremod/blocks/BlockTimberFrame.java

```diff
@@ -0,0 +1,67 @@
+package com.minecolonies.coremod.blocks;
+
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import com.minecolonies.api.util.constant.Constants;
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.ItemBlock;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+/**
+ * Decorative blocks
+ */
+public class BlockTimberFrame extends Block
```

Comment:
Why is this a different Block and not just a different BlockState.
This is occupying an additional BlockID while it is not needed nor recommended,